content/projects.md

```diff
@@ -1,21 +1,157 @@
 # Projects
 
+This document describes the organization and workflow for Optakt projects, and acts as a centralized table of contents of project documentation.
+
 ## Organization
 
-TODO: Explain what is the minimal documentation required for any Optakt project.
-TODO: Explain what how we handle CI and releases with GitHub Actions.
+Any Optakt project should have at least the following:
```

Comment:
```suggestion
Any Optakt project should at least have the following:
```

Reply:
Both are correct and I personally prefer the form that I used :p